faq-owo.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: page
-nav_order: 6
```

Comment:
Can you change the page URL to remove FAQ? It also contains links and such. Also maybe create a parent page of "Applications and Tools" or something, because there will be more stuff like Wrangler and such in the future.

Reply:
why remove faq?